crates/rpc-types-engine/src/attributes.rs

```diff
@@ -26,6 +26,11 @@ pub struct OptimismPayloadAttributes {
         serde(skip_serializing_if = "Option::is_none", with = "alloy_serde::quantity::opt")
     )]
     pub gas_limit: Option<u64>,
+    /// If set, this sets the EIP-1559 parameters for the block.
+    ///
+    /// Prior to Holocene activation, this field should always be [None].
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
+    pub eip_1559_params: Option<FixedBytes<8>>,
```

Comment:
this is u8 and can use alloy_serde::quantity::opt

Reply:
or is it?
maybe not, if this is a hash type then this should be `B8`

Reply:
It's not? It's not - it's a fixed bytes type of `8` bytes in len.

extension is spec'd as:
```
PayloadAttributesV3: {
    timestamp: QUANTITY
    random: DATA (32 bytes)
    suggestedFeeRecipient: DATA (20 bytes)
    withdrawals: array of WithdrawalV1
    parentBeaconBlockRoot: DATA (32 bytes)
    transactions: array of DATA
    noTxPool: bool
    gasLimit: QUANTITY or null
    eip1559Params: DATA (8 bytes)
}
```

we're packing 2x 32 bit values (denominator, elasticity) within it.

Reply:
changed to `B64`.

Reply:
gotcha